platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -222,7 +222,7 @@ public SwirldsPlatform(@NonNull final PlatformComponentBuilder builder) {
 
         platformWiring = new PlatformWiring(platformContext, blocks.model(), blocks.applicationCallbacks());
 
-        registerAddressBookMetrics(platformContext.getMetrics(), currentAddressBook, selfId);
+        registerAddressBookMetrics(platformContext.getMetrics(), currentRoster, selfId);
```

Comment:
Minor: might be worth renaming the method too.

Reply:
renamed the method